src/components/credit-card-date/scripts/build-dev.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
this file can be removed as "dev" builds are no longer supported, use docs instead